### DISCUSSION TITLE
Remove password confirmation in {{ user:register_form }}

### DISCUSSION THREAD

**Discussion by jevgenijs-zaremba** [statamic/cms#7276] [Q&A]
I have implemented registration form using examples described in documentation: https://statamic.dev/tags/user-register_form
Everything works fine. My only question is how to remove the requirment for the password confirmation?

**Comment by duncanmcclean** [comment]
It looks like the requirement for a password being confirmed is currently hard coded in Statamic, see https://github.com/statamic/cms/blob/3.3/src/Http/Controllers/UserController.php#L66

However, it's a [good feature request](https://github.com/statamic/ideas/issues/new)!

**Reply by jevgenijs-zaremba** [reply]
Feature request created #https://github.com/statamic/ideas/issues/933

**Comment by jevgenijs-zaremba** [comment]
As temporary fix I have solved this by using Alpine bindings. If anyone interested here is a code example:

```
<div x-data="{ password: '' }">
  {{ user:register_form redirect="/app" }}
        ...
        <label>Password</label>
        <input x-model="password" type="password" name="password" />
     
        <div class="hidden">
            <label>Password Confirmation</label>
            <input x-model="password" type="password" name="password_confirmation"/>
        </div>
        ...
  {{ /user:register_form }}   
</div>    
```
